rasterio/_filepath.pxd

```diff
@@ -0,0 +1,6 @@
+include "gdal.pxi"
+
+cdef int install_filepath_plugin(VSIFilesystemPluginCallbacksStruct *callbacks_struct)
+cdef void uninstall_filepath_plugin(VSIFilesystemPluginCallbacksStruct *callbacks_struct)
```

Comment:
Note that we're not calling this anywhere at the moment, just as we're not de-registering GDAL drivers.